schema.yaml

```diff
@@ -105,14 +156,65 @@ variables:
     description: Number of VMs to provision
     type: number
     required: true
-    default: 1
+    default: 2
```

Comment:
Please revert to 1 instance as default as the LiveLab goes over provisioning 1 node at first and then scaling.
I'll need to modify the lab to make use of the deployment here.

Reply:
Associated [Reference Architecture](https://docs.oracle.com/en/solutions/wildfly-oci/index.html) defines two JBoss nodes as a part of the topology. Spread between two availability domains (or FDs in a single AD region) will make this setup highly available, and that is why we are adding LB on top. Also, that is why I have added two nodes as default.

Reply:
I'll have to modify the LiveLab to change the default back to 1 so it works, but fine.